backend/organization/views/project_views.py

```diff
@@ -79,9 +81,13 @@ def get_queryset(self):
         if 'category' in self.request.query_params:
             project_category = self.request.query_params.get('category').split(',')
             project_tags = ProjectTags.objects.filter(name__in=project_category)
+            # Use .distinct to dedupe selected rows. We
+            # must use order_by in conjunction with distinct:
```

Comment:
When not passing a database column to distinct, we also don't need to use `order_by`